### PR TITLE
Enable to run tests for orc file format

### PR DESCRIPTION
**Pull request by lee-lei**
The changes are below:
1. Add the strategy to fix assembly error for guava package in OAP 0.5.0 release
2. Add orc configuration set for bitmap and btree test suites.

### CODE REVIEW

**Comment by lee-lei** [comment]
With oap 0.5.0 package, the commands of "sbt clean" and "sbt assembly" work well. With the generated orc data, the tests run well.

File: src/main/scala/org/apache/spark/sql/BenchmarkConfig.scala [hunk 1]
```diff
@@ -154,9 +154,30 @@ trait ParquetOnlyConfigSet extends BenchmarkConfigSelector{
...
+      .setBenchmarkConf(BenchmarkConfig.FILE_FORMAT, "orc")
+      .setBenchmarkConf(BenchmarkConfig.INDEX_ENABLE, "true"),
+    new BenchmarkConfig()
+      .setBenchmarkConfName("Orc w/o index")
+      .setBenchmarkConf(BenchmarkConfig.FILE_FORMAT, "orc")
+      .setBenchmarkConf(BenchmarkConfig.INDEX_ENABLE, "false")
+  )
+}
+
+trait ParquetOapOrcConfigSet extends BenchmarkConfigSelector{
```
Comment (JkSelf) [inline src/main/scala/org/apache/spark/sql/BenchmarkConfig.scala]: Better change the ` ParquetOapOrcConfigSet` to `ParquetVsOapVsOrcConfigSet`.

Comment (lee-lei) [reply src/main/scala/org/apache/spark/sql/BenchmarkConfig.scala]: Sure. No problem.

**Comment by JkSelf** [comment]
Thanks @lee-lei  LGTM

**Pull request merged**